examples/tests/cancelonsetup.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
```

Comment:
Well moving the `skip` to `cancel` would make it clearer, but this is also understandable...